packages/genesys-spark-components/src/components/beta/gux-rich-text-editor/gux-rich-text-editor-beta.tsx

```diff
@@ -0,0 +1,95 @@
+import { Component, h, JSX, Element, Prop } from '@stencil/core';
+import { Editor } from '@tiptap/core';
+import StarterKit from '@tiptap/starter-kit';
+import Underline from '@tiptap/extension-underline';
+import Link from '@tiptap/extension-link';
```

Comment:
I think we're looking for a dumber component that doesn't have Tiptap baked in in-case an alternative library needs to be used. The examples can use TipTap and be a preferred library, but we don't want to tie the component itself to Tiptap.